docs/editors/vim.md

```diff
@@ -17,7 +16,8 @@ which will provide the most complete implementation of LSP and Metals specific h
 ## Installing Vim
 
 The coc.nvim plugin requires either **Vim >= 8.1** or **Neovim >= 0.3.1**. Make
-sure you have the correct version installed.
+sure you have the correct version installed. While it works with both Vim and Neovim, the
+experience is a bit smoother with Neovim.
```

Comment:
I'm fine with recommending neovim over vim. How about?
```suggestion
While it works with both Vim and Neovim, we recommend using Neovim since it provides the most feature-complete experience with Metals.
```

We might want to list what features are currently working more smoothly in neovim. You never know, maybe somebody will take on the heroic task of improving the situation for vim.

Reply:
The only thing I don't like about this change is that technically it doesn't provide a more _feature-complete_ experience, since they both allow for the same features. However, in Neovim, they seem to just work smoother without any hangs. Some of it is purely performance related.

Reply:
I'll massage this a bit though.